src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -266,7 +280,7 @@ private String waitFor() throws InterruptedException {
             Logger.debug(
                 this, "#waitFor(): process finished: %s", this.process
             );
-            if (!done.await(2L, TimeUnit.SECONDS)) {
```

Comment:
only here or everywhere where N_MONITORS constant is introduced ?
2L looks to me as magic number, that's why I've made the replacement.